src/components/Lane/Lane.tsx

```diff
@@ -32,14 +36,25 @@ interface LaneItemsProps {
 function LaneItems({
   isGhost,
   items,
+  lane,
```

Comment:
@pjeby Curious about adding the lane as a prop here. If I'm reading this correctly, it's only used to grab `shouldMarkItemsComplete` which is already passed in as a prop below?

Reply:
You're right -- that was a dropped edit, as in I added shouldMarkdItemsComplete to get rid of lane, then apparently got distracted.  Fix pushed.